buildenv/jenkins/jobs/infrastructure/omrMirror

```diff
@@ -34,44 +34,41 @@ pipeline {
             steps {
                 timestamps {
                     script {
-                        withCredentials([usernamePassword(credentialsId: 'b6987280-6402-458f-bdd6-7affc2e360d4', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
-                            if (!fileExists('omr/HEAD')) {
-                                sh "git clone --mirror ${HTTP}${SRC_REPO} omr"
-                            } else {
-                                dir('omr') {
-                                    sh 'git remote update --prune'
-                                }
-                            }
+                        if (!fileExists('omr/HEAD')) {
+                            sh "git clone --mirror ${HTTP}${SRC_REPO} omr"
+                        } else {
```

Comment:
You can dispense  of the `else` clause here and move `sh 'git remote update --prune'` before the git push.

Reply:
We could, but then we would be running an extra command for nothing half of the time.

Reply:
k